R/remotebmi/README.md

```diff
@@ -0,0 +1,78 @@
+
+# remotebmi
+
+<!-- badges: start -->
+<!-- badges: end -->
+
+The goal of remotebmi is to allow a model with BMI interface written in R to be called from a Python client via a http json webservice.
+
+## Installation
+
+You can install the development version of remotebmi from [GitHub](https://github.com/) with:
+
+``` r
+# install.packages("pak")
+pak::pak("github::eWaterCycle/remotebmi/R/remotebmi")
+```
+
+## Example
+
+This is a basic example which shows you how to solve a common problem:
+
+``` r
+library(remotebmi)
+## basic example code
+
+pak::pak("github::ClaudiaBrauer/WALRUS")
+pak::pak("github::eWaterCycle/bmi-r")
```

Comment:
I get stuck on building the `stringi` package. `install.package("stringi")` fails to build, and R binaries are only available for Windows and MacOS, not linux :roll_eyes:

Reply:
Managed to finally build it with `install.packages("stringi", configure.args="--disable-pkg-config")`